ra/ra.go

```diff
@@ -264,19 +264,45 @@ func validateEmail(ctx context.Context, address string, resolver bdns.DNSClient)
 	return emptyDNSResponseError
 }
 
+// certificateRequestAuthz is a struct for holding information about a valid
```

Comment:
I think you're missing "authz" between "valid" and "referenced" on the next line.

Reply:
Good catch, thanks! Fixed in 3163976